.travis.yml

```diff
@@ -25,8 +25,8 @@ before_install:
    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository ppa:duggan/bats --yes; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get -qq update; else brew update; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install -y libusb-1.0-0-dev bats; else brew install libusb bats; fi
-   - travis_wait cargo install rustfmt --force || true
-   - travis_wait cargo install rustfmt --force || true
+   - travis_wait cargo install rustfmt-nighlty --force || true
```

Comment:
And what is this `travis_wait`? Possibly rel https://github.com/ethereumproject/go-ethereum/issues/434?

Reply:
@whilei, rustfmt currently migrating to new interface so we forced to use nightly, 
But I can't figure it out why travis uses old `rustfmt`, even when config says to use nightly.
Will manually edit this PR, and look further

Reply:
Interesting. Note also should `s/nighlty/nightly/g`